### ISSUE TITLE
Settings array empty

### ISSUE THREAD

**Issue by isvaljek** [hlashbrooke/WordPress-Plugin-Template#37]
On init, the instance of this plugin contains the object WordPress_Plugin_Template_Settings, but its field 'settings' is an empty array.
Is it my responsibility to create/load this array with plugin settings or should this happen automagically?

**Comment by geotonics** [comment]
WordPress_Plugin_Template_Settings already has settings which are automatically added by the settings_fields() function,  These settings are only examples. You will need to edit them to create the setting fields that will work for your plugin.

**Comment by isvaljek** [comment]
Yeah I created the new settings fields, and they save to db ok. But I don't know whether there's a simpler way than get_option('prefix_fieldnam') for each field.
Is it possible to get an array of all the settings at once?

**Comment by geotonics** [comment]
You can get all options with https://codex.wordpress.org/Function_Reference/wp_load_alloptions
but I don't know of any built in way to get all options for one plugin. 
You could do it by copying that function and modifying the query. .
